egs/grid/VSR/local/compute_avhubert_grid.py

```
@@ -0,0 +1,508 @@
+    device    = _worker_globals["device"]
+    
+    FIXED_DURATION: float = 3.0
+    FRAME_SHIFT: float = 0.04
+    EXPECTED_FRAMES: int = 75
+    TEMPORAL_DIM: int = 0
+
+    h5_path = feats_dir / f"{partition}_avhubert_{worker_id:02d}.h5"
+    video_cuts = []
+
+    with NumpyHdf5Writer(h5_path) as writer:
+        for recording, supervision in zip(recordings_subset, supervisions_subset):
```

Ruff (0.15.11)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)